### PR TITLE
Traditional Japanese blue collection 01

### PR DESCRIPTION
**Pull request by kyosuke25**

### CODE REVIEW

File: collections/trad-japanese-blue/inscriptions.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
+[
+  {
+    "id": "cd4fd42952aa0f7ae5ef6697d13081637b8e84b76deb69d0dc2778cc269b5de0i0",
+    "meta": {
+      "id": 1,
+      "creator": "0xKyosuke",
+      "name": "み空色 -Misorairo-",
```
Comment (MillHaus33) [inline collections/trad-japanese-blue/inscriptions.json:7]: hey i didnt notice this on the red collection but can you follow the format of metadata in the readme

**Comment by MillHaus33** [comment]
use trait type and value, and please update red if not already following format

**Comment by kyosuke25** [comment]
I see.
I'm going to update soon.
My red collection has just "name" property.

**Comment by MillHaus33** [comment]
these are cool how can i get one

**Comment by kyosuke25** [comment]
ty!
I can see my new collection.

do you want it really lol?

**Pull request merged**